index.html

Monika452 opened the pull request:
Stworzenie szablonu strony

Monika452 committed:
Create strona

Stworzenie pierwszego szablonu strony

Monika452 committed:
Zmiana nazwy

Nircek commented on index.html:
```diff
@@ -0,0 +1,10 @@
+<html>
```
w3c: **Error**: Start tag seen without seeing a doctype first. Expected `<!DOCTYPE html>`.

Nircek commented on index.html:
```diff
@@ -0,0 +1,10 @@
+<html>
+<head>
```
w3c: **Error**: Element head is missing a required instance of child element title.

Nircek commented on index.html:
```diff
@@ -0,0 +1,10 @@
+</head>
+<body>
+<div id="zawartosc">
+<div id="tytul"></div>
```
Mogłaś tutaj dodać roboczo tytuł. :stuck_out_tongue:

Monika452 committed:
poprawienie kodu

dodanie doctype i title

Nircek commented on index.html:
```diff
@@ -0,0 +1,14 @@
+<html>
+<head>
+  <title>
+  </title>
```
w3c: **Error**: Element title must not be empty.

Nircek committed:
Dodanie podstawowych rzeczy do pliku HTML
html lang
meta charset, viewport, X-UA-Compatible
wstawienie tytułu strony oraz tytułu na stronie
dodanie podstawowego CSSa do wyśrodkowywania
poprawa tabulacji
closes #7

Monika452 committed:
Dodanie nowych divów

Divy na logo i autorów

Monika452 committed:
Create autorzy.html
